code/modules/antagonists/horror/horror_html.dm

```diff
@@ -0,0 +1,102 @@
+/mob/living/simple_animal/horror/proc/get_html_template(content)

```

Comment:
sad TGUI noises

Reply:
i would do it into TGUI but apparently i can't make TGUI work while a mob is inside another mob

Reply:
Try harder